Helper/CustomsConfig.php

```diff
@@ -33,6 +33,17 @@ class CustomsConfig extends AbstractHelper
 
     const PRODUCT_ATTR_CROSS_BORDER_TYPE = 'avatax_cross_border_type';
 
+    /**
```

Comment:
Switch to using this DocBlock syntax:

```
    /**#@+
     * Module config settings
     */
     …
     /**#@-*/
```

See this for reference: https://github.com/classyllama/ClassyLlama_AvaTax/blob/1c1e7122fc05b0642b683ada17b1125ec7218f41/Helper/Config.php#L36..L146